ote/src/cljs/ote/views/route/trips.cljs

```diff
@@ -136,20 +138,25 @@
 (defn trips-list [e! route]
   (let [stop-sequence (::transit/stops route)
         stop-count (count stop-sequence)
-        trips (::transit/trips route)]
-
+        trips (::transit/trips route)
+        empty-calendar? (empty? (first (:ote.db.transit/service-calendars route)))]
     [:div.route-times
      [:table {:style {:text-align "center"}}
       [route-times-header stop-sequence]
       [:tbody
        (doall (map-indexed (partial trip-row e! stop-count) trips))]]
+
+     (when empty-calendar?
+       [:div {:style {:margin-left "25px" :margin-right "25px" :margin-top "10px"}}
```

Comment:
Another opinion; the help box looks better if you remove margin-left and margin-right.

Reply:
Fixed.